spec/models/instance_spec.rb

```diff
@@ -3,7 +3,10 @@
 RSpec.describe Instance, type: :model do
   it { is_expected.to have_many(:instance_users) }
   it { is_expected.to have_many(:users).through(:instance_users) }
-  it { is_expected.to have_many(:announcements).class_name(Instance::Announcement.name) }
+  it do
+    is_expected.to have_many(:announcements).class_name(Instance::Announcement.name).
```

Comment:
`is_expected` is used only when the whole block fits on one line. Adding `do` makes it weirder.